t/data/workers.ini

```diff
@@ -3,6 +3,7 @@
 [global]
 HOST = http://localhost:9526
 WORKER_HOSTNAME = 127.0.0.1
+CRITICAL_LOAD_AVG_THRESHOLD = 0
```

Comment:
I suggest you add a comment here explaining context and the reason

Reply:
I've added a comment above the variable, what do you think?

Reply:
nice!